tools/hlslpm/github.py

```diff
@@ -146,6 +236,13 @@ def read_file_relative_to_this_script(filename):
             print(f"Title: {i.title}")
             print(f"Body: {i.body}")
             print('-----')
-            # Serialize 'issues' list to JSON
-            json_data = json.dumps([i.__dict__ for i in issues])
-            print(json_data)
+        # Serialize 'issues' list to JSON
+        json_data = json.dumps([i.__dict__ for i in issues])
+        print(json_data)
+
+    if False:
+        r = gh.get_issues_tracked(["I_kwDOMbLzis6Rpmkm", "I_kwDOBITxeM5SPFcF"])
+        print(list(r))
+
+    r = gh.get_project_field_info("ProjectMilestone")
+    print(r)
```

Comment:
This is an interesting comment/debugging style

Reply:
Yeah, it's helpful to keep these tests around, but I don't want to accidentally run them against a live github.  But I also don't want to spend the time to build a mock github to test against.